packages/core/src/card/card.element.spec.ts

```diff
@@ -0,0 +1,26 @@
+import '@cds/core/card/register.js';
+import { html } from 'lit-html';
+import { componentIsStable, createTestElement, removeTestElement } from '@cds/core/test';
+import { CdsCard } from '@cds/core/card';
+
+describe('Card element: ', () => {
+  let testElement: HTMLElement;
+  let component: CdsCard;
+  const placeholderText = 'Placeholder';
+
+  afterEach(() => {
+    removeTestElement(testElement);
+  });
+
+  describe('create the card component', () => {
```

Comment:
This describe isn't needed, move the `beforeEach` up to the top level describe and eliminate this nested one.